make-classic.sh

```diff
@@ -1,13 +1,5 @@
 #!/bin/sh
 
-sed -i 's/base: core22/base: core20/g' snap/snapcraft.yaml
```

Comment:
@simskij @dstathis why did we build the classic snap on core20?
Tested manually on core22 and seems ok.